scheduler/scheduler.go

```diff
@@ -848,7 +838,7 @@ func (s *scheduler) walkWorkflow(prnodes []*processNode, pbnodes []*publishNode,
 		} else {
 			depGroupMap[pr.Target] = depGroup{Plugins: []core.SubscribedPlugin{pr}}
 		}
-		s.walkWorkflow(pr.ProcessNodes, pr.PublishNodes, depGroupMap)
+		getWorkflowPlugins(pr.ProcessNodes, pr.PublishNodes, depGroupMap)
```

Comment:
I don't think it may cause any problems. Gathered plugins are used for pool subscriptions which is idempotent operation.